resources/en-US/endpoints.bxb

```diff
@@ -0,0 +1,8 @@
+endpoints {
```

Comment:
Did you mean to add this file as part of this?

Reply:
Actually, I had mistakenly created to locale folders, so I've removed one. Thanks for catching that.